.actions/helpers.py

```diff
@@ -148,6 +148,10 @@ class HelperCLI:
     REQUIREMENTS_FILE = "requirements.txt"
     PIP_ARGS_FILE = "pip_arguments.txt"
     META_PIP_KEY = "pip__"
+    TAGS = {
+        "course_UvA-DL": "UvA-DL-Course",
```

Comment:
@ethanwharris where is the name with spaces instead of the -
Maybe I just don't see it =)

Reply:
Haha, the dashes get turned into spaces in the tag view thing. So `course_UvA-DL` becomes `course_UvA DL` and `UvA-DL-Course` becomes `UvA DL Course`.

Reply:
I see so lest add this as comment to the code